src/app/core/components/navigation/template.html

```diff
@@ -26,6 +26,7 @@
     </button>
     <a class="km-header-logo"
        routerLink="projects"></a>
+    <div class="km-text">{{currentView}}</div>
```

Comment:
remove extra space `"fxLayoutAlign=" center"` in L# 19 can't comment just mentioning here
- space between " <SPACE> center"

Reply:
Not sure if you know it or not, but `"fxLayoutAlign=" center"` will have different result than `"fxLayoutAlign="center"`. One will center horizontally, one vertically. Empty space is to ignore one of the params.